src/middlewared/middlewared/plugins/disk_/disk_info.py

```diff
@@ -7,6 +9,65 @@
 from middlewared.service import CallError, private, Service
 
 
+# The basic unit of a block I/O is a sector. A sector is
+# 512 (2 ** 9) bytes. In sysfs, the files (sector_t type)
+# `<disk>/<part>/start` and `<disk>/<part>/size` are
+# shown as a multiple of 512 bytes. Most user-space
+# tools (fdisk, parted, sfdisk, etc) treat the partition
+# offsets in sectors.
+BYTES_512 = 512
+PART_INFO_FIELDS = (
+    # queue/logical_block_size (reported as a multiple of BYTES_512)
```

Comment:
If you want default values and documentation, you’ll be better off using a dataclass